app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,12 @@
+class FoodsController < ApplicationController
+    def index
+        @food = FoodFacade.search_list_by_food(params[:search])
+
+    end
+
+    private
+
+    def food_params
```

Comment:
Strong params are needed when we are gathering user input from something like a form. In this case, are strong params needed for name, brand, code, ingredients?